project/ProjectPlugin.scala

```diff
@@ -169,8 +170,8 @@ object ProjectPlugin extends AutoPlugin {
     )
 
     lazy val noCrossCompilationLastScala: Seq[Def.Setting[_]] = Seq(
```

Comment:
**suggestion**: As we are not cross-compiling with any other Scala version, we could remove these settings.

Reply:
yup, sounds good!